contrib/init/lnd.service

```diff
@@ -0,0 +1,50 @@
+# A sample systemd service file for lnd running with a bitcoind service.
+
+[Unit]
+Description=Lightning Network Daemon
+
+# Make sure lnd starts after bitcoind is ready
+Requires=bitcoind.service
+After=bitcoind.service
+
+[Service]
+ExecStart=/usr/local/bin/lnd
+ExecStop=/usr/local/bin/lncli stop
+PIDFile=/home/bitcoin/.lnd/lnd.pid
+
+# Replace these with the user:group that will run lnd
+User=bitcoin
+Group=bitcoin
+
+# Try restarting lnd if it stops due to a failure
+Restart=on-failure
```

Comment:
I'd recommend using `always` in case `lnd` accidentally exits with code 0. It can still be stopped by explicit `systemctl` command, so no harm there and most people probably want their channels to be watched as reliably as possible.

Reply:
Would it restart automatically if you used `lncli stop` with `always`?

Reply:
No, `systemctl stop` will keep it stopped which I believe is good. One could add `RefuseManualStop` to avoid stopping it accidentally but IMO that's too much.

Reply:
Yes for `systemctl stop` but what about `lncli stop` or otherwise shutting down the node via the gRPC layer? I figure ideally both should actually stop it  but I assumed the latter would not stop it for good with `Restart=always`.

Reply:
Sorry, misread. Yes, `lncli stop` wouldn't work. I personally don't like stopping thing via RPC but I can understand that people would find it surprising to not work, so not pushing it.